### PR TITLE
UIDS 29 Add all the colors and export them.

### PR DESCRIPTION
**Pull request by rsaris**
Also included exporting FadeTransition.

### CODE REVIEW

File: scss/palette.scss [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 $ux-black: #000000;
 $ux-blue: #337ab7;
-$ux-dark-blue: #639BB8;
```
Comment (rsaris) [inline scss/palette.scss]: We don't use this anywhere so removed it.

File: stories/Palette.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import palette from '../src/Styles/palette';
+
+import { withPadding } from './decorators';
+
+const renderColor = (colorName) => (
+  <div style={{ display: 'flex', height: '20rem', width: '100%' }}>
```
Comment (rsaris) [inline stories/Palette.stories.jsx]: Not sure how we feel about using styles like this in stories but I figured it was better than trying to interject scss files into the stories directory.

File: src/index.js [hunk 3]
```diff
@@ -11,17 +12,21 @@ import Pill from './Pill/Pill';
 import Popper from './Popper/Popper';
 import RadioButton from './RadioButton/RadioButton';
 import RadioButtonGroup from './RadioButtonGroup/RadioButtonGroup';
+import palette from './Styles/palette';
```
Comment (davidmferris) [inline src/index.js]: Wondering if we should make a /Styles/index.js file and import like `import { palette, zStack } from 'ui-design-system/Styles` to differentiate it from components. What do you think? FYI as of v1.1.0 the /Styles directory is included in the package lib

File: stories/Palette.stories.jsx (hunk 2 above)
Comment (davidmferris) [reply stories/Palette.stories.jsx]: This is cool with me. Do you think it'd be helpful to add the color names (e.g. `uxBlue200`) somewhere on or near each swath for anyone who needs to quickly cross reference?

File: src/index.js (hunk 3 above)
Comment (rsaris) [reply src/index.js]: I'm into it.

File: stories/Palette.stories.jsx (hunk 2 above)
Comment (rsaris) [reply stories/Palette.stories.jsx]: DONE!

**Pull request merged**